### DISCUSSION TITLE
Governor: Acala parameters

### DISCUSSION THREAD

**Discussion by tbjump** [wormhole-foundation/wormhole#2609] [Governance]
Over the last week, there has been a significant increase in transfer activity from DAI on Acala to Ethereum. 

|    date    | notional_transferred (dai)|
------------|-----------------------
 2023-04-02 | 8720
 2023-04-03 | 8662
 2023-04-04 | 9590
 2023-04-05 |    20304
 2023-04-06 |   642799
 2023-04-07 |   357062
 2023-04-08 | 9993
 2023-04-10 |   467001

Looking at the [Dashboard](https://wormhole-foundation.github.io/wormhole-dashboard/), the Governor "fast transaction limit" is currently exhausted with five VAAs already enqueued.

Raising the Governor limit would expose the Token Bridge to a higher security risk if there was a vulnerability on the Acala blockchain, but since the launch of the [Global Accountant](https://github.com/wormhole-foundation/wormhole/pull/2406/files), the cross-chain risk is limited. 

Acala has received [multiple third-party audits](https://medium.com/acalanetwork/acala-security-audit-edd1850e27aa) and runs a [bug bounty program](https://immunefi.com/bounty/acala/). 

From a security perspective, I would support raising the Governor fast-transaction limit from $500/$50k to $1M/$100k notional to meet user demand and re-assess the situation if demand falls back down.

I would appreciate feedback from Guardians if they would adopt a $1M/$100k setting for Acala if it were to become part of the reference implementation, or alternatively which limits they seem to be appropriate.

**Comment by dancamarg0** [comment]
I think we should support users demand whenever security isn't a problem. It feels secure to us (Triton) to raise this limit.

**Comment by tbjump** [comment]
It seems like demand on Acala went back to normal and a modification of Governor parameters is no longer needed. I will therefore close this discussion for now.
